java/src/jmri/LightManager.java

```diff
@@ -203,4 +203,9 @@
     @CheckReturnValue
     public boolean allowMultipleAdditions(@Nonnull String systemName);
 
+    /**
+     * Provide a connection specific tooltip and entry validation regex for the Add new item beantable pane.
+     */
+    public String[] getAddFormat();
```

Comment:
This needs to be rethought. You really should not return disparate items (a tooltip and a regex in this case) in a single array.